3-extra/1-count-words.js

```diff
@@ -25,11 +25,26 @@
 
 function countWords(string) {
   const wordCount = {};
+  const words = string.split(" ")
 
```

Comment:
this looks really good but you could also check if the string is empty before beginning?

Reply:
you're also only splitting using spaces, what if there was a comma or any other punctuation in the string would it still work?